apps/dashboard/src/api/workflows.ts

```diff
@@ -27,16 +27,31 @@ export const getWorkflows = async ({
   limit,
   query,
   offset,
+  orderByField,
+  orderDirection,
 }: {
   environment: IEnvironment;
   limit: number;
   offset: number;
   query: string;
+  orderByField?: string;
+  orderDirection?: string;
 }): Promise<ListWorkflowResponse> => {
-  const { data } = await getV2<{ data: ListWorkflowResponse }>(
-    `/workflows?limit=${limit}&offset=${offset}&query=${query}`,
-    { environment }
-  );
+  const params = new URLSearchParams({
+    limit: limit.toString(),
+    offset: offset.toString(),
+    query,
+  });
+
+  if (orderByField) {
+    params.append('orderByField', orderByField);
```

Comment:
```suggestion
    params.append('orderBy', orderBy);
```

Reply:
Why? this is not how our API expect this
![CleanShot 2025-01-22 at 10 26 47@2x](https://github.com/user-attachments/assets/d55a3255-9352-4759-8efe-f41fd125bc69)

Reply:
The DTO that you mentioned is the right one but only used from Workflow V2 endpoints for now. So there is no risk of breaking changes. The orderBy is fewer characters, and the field feels kind of redundant. It's like naming a field `createdAtDate` instead of `createdAt`.

Also, some legacy V1 controllers, such as layout, use the `orderBy` parameter.